### ISSUE TITLE
Q: Is it correct behaviour to still download .srt files when "embedded in video" is enabled?

### ISSUE THREAD

**Issue by ghost** [Ascoware/get-iplayer-automator#358]
**What steps will reproduce the problem? Be specific, and provide as much detail as possible.**
Any download with "**download subtitles**" and "**Embed Subtitles**" **checked**

**What is the expected output? What do you see instead?**
Embedded subtitles only (no separate .srt file)

**What version of GiA are you using? On what version of OS X?**
1.21.12

**Are you using a web proxy? DNS proxy? VPN? VPS?**
No

**What program are you attempting to download (full title, series and episode)? **Provide only ONE program as an example.** **
Fort Salem - s02e01

**Comment by skovatch** [comment]
It looks like at some point in the app's history there was an option to keep or delete the raw srt files, but I can't find it in the preferences anymore. The code is still there, so I guess it was dropped before I took over. I can look into adding it back.

**Comment by jchhenderson** [comment]
thanks, guys!

**Comment by hugofalkman** [comment]
@skovatch Today's release 1.21.13 has a line item "subtitles: Always delete the raw subtitle files". As far as I can see, the GiA behavior is unchanged: the .srt file is deleted for ITV, but not for BBC.

**Comment by skovatch** [comment]
> @skovatch Today's release 1.21.13 has a line item "subtitles: Always delete the raw subtitle files". As far as I can see, the GiA behavior is unchanged: the .srt file is deleted for ITV, but not for BBC.

You're right, I made a change but it was a while ago and I didn't really test it. This might be tough to fix because get_iplayer no longer prints out the path to the .srt file so I can't catch it for cleanup afterwards.

**Comment by hugofalkman** [comment]
@skovatch Don't know get_iplayer, but if it's GiA that sends the mp4 file to the iTunes library, the srt file is in the same folder. Btw that's true for youtube-dl as well.

**Comment by ejstubbs** [comment]
Although I agree it's a bit inconvenient to have unwanted srt files floating about, in my case at least I actually need both srt files and embedded subtitles.  I watch videos off my NAS on a number of different devices using a variety of different streaming clients.  Some of those clients can't handle embedded subtitles, and need the separate srt file.  Others can only use embedded subtitles and ignore srt files.  Having both available for a given programme means that I've got useful flexibility in my viewing choices.  So by all means offer the _option_ to delete the srt file, but the option _not_ to delete should also be retained.  If that option meant that srt files from ITV could also be retained then that would be a significant improvement for me, because at the moment I am restricted as to which clients I can watch ITV programmes on if I want subtitles.  (And yes, I know there are ways to extract srt files from video with embedded subtitles, but it's a bit of a hit and miss process, and if the srt files are retained after downloading that wouldn't be necessary.)
